antarest/study/business/utils.py

```diff
@@ -40,12 +42,23 @@ def execute_or_add_commands(
 ) -> None:
     if isinstance(study, RawStudy):
         executed_commands: t.MutableSequence[ICommand] = []
+        should_invalidate_cache = False
         for command in commands:
+            if not command.can_update_study_config():
```

Comment:
Reading the code, I am thinking that maybe it would be even better to have the "should_invalidate" in the command output instead.
It seems to make more sense, because it's the command application which implements and knows if the config has been updated or not, so it would make sense to return this information with the command application result.

Reply:
Note that we'll need to adapt study interfaces now ...